app/views/users/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<p id="notice"><%= notice %></p>
+
+<div class="wrapper">
+  <div style="float: right">
```

Comment:
What's the thinking behind the inline styling here, as opposed to creating a class for the float behavior and putting it in a style sheet? 

This styling may not actually be applied here, I believe you need a semicolon at the end of the line for the styling to be parsed, ie, `<div style="float:right;">`